cloud/google/node_init.sh

```diff
@@ -42,12 +42,17 @@ fi
 sudo bash ./${AGENT_INSTALL_SCRIPT}
 
 # Examine what kind of Docker image is on this machine to determine what to log.
-if docker images | grep ct-server; then
+DOCKER_REPOS_FILE=$(mktemp)
+docker images --format '{{ .Repository }}' > "${DOCKER_REPOS_FILE}"
```

Comment:
Looks good, but does this require a newer Docker than we have on our VMs at present?

$ docker -v
Docker version 1.9.1, build a34a1d5

$ docker images --format '{{ .Repository }}'
flag provided but not defined: --format
See 'docker images --help'.